src/components/Responsive/Hidden/Hidden.test.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { mount } from 'enzyme';
 import Context from '../Context';
-import Hidden from './';
+import Hidden from '.';
 
 describe('<Hidden />', () => {
   it('renders without error', () => {
     mount(
-      <Hidden>
+      <Hidden breakpoints={[]}>
```

Comment:
See my comment below - this shouldn't need to be required.
I agree it doesn't make much sense to omit it, but in the spirit of robustness "be conservative in what you do, be liberal in what you accept from others"

Reply:
Postel's law, if you're not familiar. I love how it was written about TCP specs, but it's also good ethics.
https://en.wikipedia.org/wiki/Robustness_principle